.github/workflows/validate-pull-request.yml

```diff
@@ -0,0 +1,8 @@
+name: Validate Pull Request
+
+on:
+  pull_request:
+
+jobs:
+  validate-pull-request:
+    uses: Lombiq/GitHub-Actions/.github/workflows/validate-pull-request.yml@dev
```

Comment:
Please restore the usage of the same workflow as before.

Reply:
done